18F/identity-design-system
# Use USWDS icons with badge component

## 🛠 Summary of changes

Updates Badge component to use USWDS icons.

This works toward removing our custom alert icons.

Related: #439, #440

## 📜 Testing Plan

1. Run locally as npm start
2. Go to http://localhost:4000/badges/
3. Observe badge icons

## 👀 Screenshots

Before|After
---|---
![before](https://github.com/18F/identity-design-system/assets/1779930/1d5dffe1-2614-4170-aad6-d5398e681668)|![after](https://github.com/18F/identity-design-system/assets/1779930/975b0e28-6ecd-4ef8-bad4-1a1c13490659)
